externs/ngeox.js

```diff
@@ -49,3 +49,75 @@ ngeox.interaction.MeasureOptions.prototype.style;
  * @type {ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined}
  */
 ngeox.interaction.MeasureOptions.prototype.sketchStyle;
+
+
+/**
+ * Namespace.
+ * @type {Object}
+ */
+ngeox.profile;
+
+
+/**
+ * Options for the profile.
+ *  @typedef {{
+ *    extractor: ngeox.profile.ProfileExtractor,
+ *    light: (boolean|undefined),
+ *    hoverCallback: (function(Object)|undefined),
+ *    outCallback: (function()|undefined),
+ *    zKeys: (Array.<string>|undefined)
+ *  }}
+ */
+ngeox.profile.ProfileOptions;
+
+
+/**
+ * @type {ngeox.profile.ProfileExtractor}
+ */
+ngeox.profile.ProfileOptions.prototype.extractor;
+
+
+/**
+ * @type {boolean|undefined}
+ */
+ngeox.profile.ProfileOptions.prototype.light;
+
+
+/**
+ * @type {function(Object)|undefined}
+ */
+ngeox.profile.ProfileOptions.prototype.hoverCallback;
+
+
+/**
+ * @type {function()|undefined}
+ */
+ngeox.profile.ProfileOptions.prototype.outCallback;
+
+
+/**
+ * @type {Array.<string>|undefined}
+ */
+ngeox.profile.ProfileOptions.prototype.zKeys;
+
+
+/**
+ * Profile data extractor.
+ * @typedef {{
+ *   z: (function(Object, string=): number),
+ *   dist: (function(Object): number)
```

Comment:
You are right, fixed.
I added this and zKeys in anticipation of multiple elevation values
handling. However, it is better if you add it when you actually implement
it.